Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1457,6 +1458,13 @@ class IUserGroupsSettingsSchema(Interface):
     )
 
 
+def validate_twitter_username(value):
+    if value and value.startswith('@'):
+        raise Invalid(
+            u'Twitter username should not include the "@" prefix character.')
```

Comment:
Indeed, in our form framework validation and manipulation/extraction of values are two different domains. Do not change the value in a validator. It may introduce some harm.